client/src/components/Lesson/WordCreator/ContextSentences.tsx

```diff
@@ -39,13 +46,36 @@ export class ContextSentences extends React.Component<ContextSentencesProps, Con
             newVal = newVal.filter((sentence, i) =>
                 sentence !== '' || i === this.props.value.length);
             this.props.onChange(newVal);
+            if (ev.target) {
+                const search = ([] as number[]);
```

Comment:
Update syntax like above and change variable name. Using `search` and `search()` is a bit confusing.